src/confcom/azext_confcom/README.md

```diff
@@ -0,0 +1,484 @@
+
+**Examples:**
+
+Example 1: The following command creates a CCE policy and outputs it to the command line: <br /> 
+
+    az confcom acipolicygen -a .\template.json --print-policy
+
+This command combines the information of images from the ARM template with other information such as mount, environment variables and commands from the ARM template to create a CCE policy. 
+The `--print-policy` argument is included to display the policy on the command line rather than injecting it into the input ARM template.
+
+Example 2: This command injects a CCE policy into [ARM-template](arm.template.md) based on input from [parameters-file](template.parameters.md) so that there is no need to change the ARM template to pass variables into the CCE policy: <br />
+
+    az confcom acipolicygen -a .\arm-template.json -p .\template.parameters.json
+
+This is mainly for decoupling purposes so that an ARM template can remain the same and evolving variables can go into a different file. 
+
+Example 3: This command takes the input of an ARM template to create a human-readable CCE policy in pretty print JSON format and output the result to the console. 
+NOTE: Generating JSON policy is for use by the customer only, and is not used by ACI In most cases. The default REGO format security policy is required. <br />
+
+    az confcom acipolicygen -a ".\arm_template" --outraw-pretty-print --json --print-policy
+
+The default output of `acipolicygen` command is base64 encoded REGO format. 
+This example uses the --json argument to generate output in JSON format, use `--outraw-pretty-print` to indicate decoding policy in clear text and in pretty print format and use `--print-policy` to print result to console. 
+
+Example 4: The following command takes the input of an ARM template to create a human-readable CCE policy in clear text and print to console: <br />
+    
+    az confcom acipolicygen -a ".\arm-template.json" --outraw --print-policy
+
+Use `--outraw` argument to output policy in clear text compact REGO format.
+
+Example 5: Input an ARM template to create a human-readable CCE policy in pretty print REGO format and save the result to a file named ".\output-file.rego": <br />
+
+    az confcom acipolicygen -a ".\arm-template" --outraw-pretty-print --save-to-file ".\output-file.rego"
+
+Example 6: Validate the policy present in the ARM template under "ccepolicy" and the containers within the ARM template are compatible. If they are incompatible, a list of reasons is given and the exit status code will be 2: <br />
+
+    az confcom acipolicygen -a ".\arm-template.json" --diff
+
+Example 7: Decode the existing CCE policy in ARM template and print to console in clear text. 
+
+    az confcom acipolicygen -a ".\arm-template.json" --print-existing-policy --outraw
```

Comment:
Sorry, I didn't explain this tag very well. `--print-existing-policy` will always pretty print. The `--outraw` won't do anything so it should be omitted